docs(Update README): Add description and edit setup (#106)
SmartBear/swaggerhub-cli: README.md

$ npm i -g swaggerhub-cli
```
## Setup
The SwaggerHub CLI can be configured through environment variables or through the [`swaggerhub configure`](#swaggerhub-configure) command. The CLI will look for the following environment variables.

> * `SWAGGERHUB_API_KEY` (required) – **Important: please keep this key secure**. This is the SwaggerHub API key the CLI will use for authentication. An API key may be [generated from the user settings page in SwaggerHub](https://app.swaggerhub.com/settings/apiKey).

Reviewer: We don't need to generate the key, it's already there.
```suggestion
* `SWAGGERHUB_API_KEY` (required) – **Important: keep this key secure.** This is the SwaggerHub API key the CLI will use for authentication. You can find your API key on the [user settings page](https://app.swaggerhub.com/settings/apiKey) in SwaggerHub.
```

> * `SWAGGERHUB_URL` (optional, default is `https://api.swaggerhub.com/`) – Customers with on-premise installations can set to point to their on-premise API, which is `http(s)://{swaggerhub-host}/v1` (do not append a backslash). 

Reviewer: ```suggestion
* `SWAGGERHUB_URL` (optional, default is `https://api.swaggerhub.com/`) – Customers with on-premise installations need point this to their on-premise API, which is `http(s)://{swaggerhub-host}/v1` (do not append a backslash). 
```

Author: `need to point this`?

Reviewer: @conor-ward yes, thanks

Alernatively, you can use the `swaggerhub configure` command to create a configuration file for the CLI to use. This command will walk you through the steps to set up the necessary configurations.

```sh-session
$ swaggerhub configure
? SwaggerHub URL: https://api.swaggerhub.com
? API Key: <your-api-key>
```

> Environment variables will take precedence over the configuration file created by this command.

Reviewer: ```suggestion
Environment variables take precedence over the configuration file created by this command.
```

Author: Sorry, missed this one
